index.js

```diff
@@ -690,7 +692,7 @@ function Physics (mcData, world) {
         vel.y += 0.04
       } else if (entity.onGround && entity.jumpTicks === 0) {
         const blockBelow = world.getBlock(entity.pos.floored().offset(0, -0.5, 0))
-        vel.y = Math.fround(0.42) * ((blockBelow && blockBelow.type === honeyblockId) ? physics.honeyblockJumpSpeed : 1)
+        vel.y = 0.42 * ((blockBelow && blockBelow.type === honeyblockId) ? physics.honeyblockJumpSpeed : 1)
```

Comment:
This change is incorrect. Current version is right.

See https://github.com/extremeheat/extracted_minecraft_data/blob/client1.19.3/client/net/minecraft/world/entity/LivingEntity.java#L2019 for reference.